instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -16,11 +16,22 @@
 Some utils used by the redis integration
 """
 
+from typing import Any
```

Comment:
```suggestion
from __future__ import annotations

from typing import Any
```
I think we'll need this

Reply:
Thank you! I only ran 313 tests. This time, I ran all tests (3.8, 3.9 ... 3.13) and linter -- All pass.